server/coordinator.go

```diff
@@ -109,6 +111,21 @@ func (c *coordinator) dispatch(region *RegionInfo) *pdpb.RegionHeartbeatResponse
 }
 
 func (c *coordinator) run() {
+	ticker := time.NewTicker(runSchedulerCheckInterval)
+	defer ticker.Stop()
+	for {
+		if c.shouldRun() {
+			break
+		}
+		log.Info("coordinator: Waiting for the cluster information to be collected")
```

Comment:
what I mean is to add log before `for` and before `break`. 
No need to log here every time.